### DISCUSSION TITLE
youtube

### DISCUSSION THREAD

**Discussion by anrulev** [xvzc/SpoofDPI#84] [Q&A] [answered]
Hi!
What regexp might be suitable to address youtube? Or maybe there is a possibility to specify a domain instead of URL in the parameters? So far, all experiments have failed.
I use --enable-doh --window-size 0

**Comment by xvzc** [comment]
Which country are you at?

**Reply by anrulev** [reply]
> Which country are you at?

Russia

**Comment by xvzc** [comment, marked as answer]
Would you try with the latest release(v0.10.4) with no option?

**Reply by xvzc** [reply]
Plus, is youtube basically blocked in your country?

**Reply by anrulev** [reply]
> Would you try with the latest release(v0.10.4) with no option?


v0.10.4 works without any options, thanks!

Youtube is not blocked, the speed is cut by dpi

**Reply by anrulev** [reply]
Looks like with v0.10.4 the problem of working with other apps and sites is gone, that's great!

**Reply by xvzc** [reply]
--pattern youtube.com will fragment packets only for youtube. Althogh every packets will go through the proxy server, the other URLs that does not match the pattern will be sent as regular packets.

1 question. why would you want to only enable fragmentation for youtube?

**Reply by anrulev** [reply]
> --pattern youtube.com will fragment packets only for youtube. Althogh every packets will go through the proxy server, the other URLs that does not match the pattern will be sent as regular packets.
> 
> 1 question. why would you want to only enable fragmentation for youtube?

--pattern youtube.com     -    it doesn't work. 
INFO[2024-08-03-03T11:17:49+03:00] Regex Pattern: youtube.com   

previous version caused conflicts with other sites and applications.

the new version works fine without additional parameters!
problem no longer exists

**Comment by flytaly** [comment]
Try googlevideo, this is the domain that provides videos, and which have been slowed down.

**Reply by anrulev** [reply]
> Try googlevideo, this is the domain that provides videos, and which have been slowed down.

может я как-то неправильно пишу регэксп, но все равно не помогает. В любом случае проблема отпала со свежим релизом.
